app/views/shared/_topbanner.html.erb

```diff
@@ -0,0 +1,13 @@
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
+
```

Comment:
In a partial you don't need the `<head>` tag. That's only in your layout.

Please remove it from here 😄